tests/ui_tools/test_popups.py

```diff
@@ -598,6 +598,23 @@ def test_keypress_show_msg_info(
             time_mentions=list(),
         )
 
+    @pytest.mark.parametrize(
+        "key",
+        {
+            *keys_for_command("COPY_MESSAGE"),
+        },
```

Comment:
nit: this fits on one line? It's probably only the extra comma(s) that's forcing black to spread it out.

Reply:
Agreed. Changed.